templated_output/Arkansas_Nuclear_2.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" standalone="no" ?>
```

Comment:
If this file is to be included in a master file, this header tag probably isn't supposed to be here.